moped-database/views/project_list_view.sql

```diff
@@ -1,6 +1,4 @@
--- latest version 1697768359465_dedupe_plv_tks
-DROP VIEW project_list_view;
```

Comment:
iirc there is an edge case that this handled for when replacing a view can fail. I'm going to think about it some more, but it could very well be an artifact that we keep carrying to future versions. 🤔

Reply:
Found this link, and it sounds like what I remember https://dba.stackexchange.com/questions/62813/why-is-it-not-possible-to-alter-a-view-with-a-create-or-replace

Reply:
just realized this is coming from a different PR. I'll add these comments there!

Reply:
yes! this is my fault. When i dropped the project list view, it said that i had to cascade all the other views. but since i was just adding a new column, and not removing or renaming a column, i found that i didnt have to drop the view to replace it